client/src/main/scala/org/apache/celeborn/client/commit/CommitHandler.scala

```diff
@@ -265,14 +266,29 @@ abstract class CommitHandler(
           slaveIds,
           getMapperAttempts(shuffleId),
           commitEpoch.incrementAndGet())
-        val res = requestCommitFilesWithRetry(worker.endpoint, commitFiles)
+        val res =
```

Comment:
If worker in blacklist, we can fast return commitFiles at the very beginning of the method

Reply:
> If worker in blacklist, we can fast return commitFiles at the very beginning of the method

We can't return. In fact we need to construct a ```CommitFilesResponse``` with proper ```failedMasterIds``` and ```failedSlaveIds```.

Reply:
> > If worker in blacklist, we can fast return commitFiles at the very beginning of the method
> 
> We can't return. In fact we need to construct a `CommitFilesResponse` with proper `failedMasterIds` and `failedSlaveIds`.

I see.